README.md

```diff
@@ -103,6 +103,10 @@ A `QBWC::Worker#requests` method cannot access values that are in-memory (global
 
 If requests are passed to `QBWC.add_job`, any `QBWC::Worker#requests` method will be ignored and will not be invoked during QuickBooks Web Connector sessions.
 
+### Referencing memory values when handling responses ###
+
+Similarly, a `QBWC::Worker#handle_response` method cannot access values that are in-memory at the time that `QBWC.add_job` is called; however, you can optionally pass arbitrary (serializable) data values to `QBWC.add_job`. This data will immediately be serialized and persisted by `QBWC.add_job`, then later deserialized and passed to `QBWC::Worker#handle_response` during a QuickBooks Web Connector session.
```

Comment:
Bullets 2-4 done. For bullet 1, I can be swayed but the title and content of this section flow well as-is in context of the section above it.